ansible_runner/runner_config.py

```diff
@@ -724,12 +724,30 @@ def _parse_cli_execenv_cmd_playbook_args():
 
             if 'podman' in self.process_isolation_executable:
                 # container namespace stuff
+                new_args.extend(["--group-add=root"])
                 new_args.extend(["--userns=keep-id"])
                 new_args.extend(["--ipc=host"])
```

Comment:
This code is only reached if in the block `if self.cli_execenv_cmd`. Is that intended?

I can agree (by experimentation) that if `--userns=keep-id` is used, then `--group-add=root` is needed (in addition to the Dockerfile changes).

But why is `--group-add=root --userns=keep-id --ipc=host` added for `ansible-runner playbook/adhoc` but not added for `ansible-runner run`?